pages/registration/application/[application_id].js

```diff
@@ -1,157 +1,211 @@
-                            disabled={application.status !== "submitted"}
-                            onClick={() => sendDecision("rejected")}>
-                            Reject Application
-                        </button>
-                    </form>
-                </div>
-
-            </main>
-
-            <footer className={styles.footer}>
-            </footer>
-        </div>
-    );
+    confirmDecision(msg, decision);
+  }
+
+  const convertObject = (obj) => {
+    let arr = [];
+
+    arr.push({field: "Applied Date", value: String(application.appliedDate).split('T')[0]});
+    arr.push({field: "Decided by ID", value: application.decidedById});
+    arr.push({field: "Name", value: application.name});
+    arr.push({field: "Date of Birth", value: String(application.dateOfBirth).split('T')[0]});
+    arr.push({field: "Blood Group", value: application.bloodGroup});
+    arr.push({field: "Birth Registration ID", value: application.birthRegistrationId});
+    arr.push({field: "Registration ID", value: application.registrationId});
+    arr.push({field: "Present Address", value: application.presentAddress});
+    arr.push({field: "Permanent Address", value: application.permanentAddress});
+    arr.push({field: "Guardian Name", value: application.guardianName});
+    arr.push({field: "Guardian Email", value: application.guardianEmail});
+    arr.push({field: "Guardian Phone", value: application.guardianPhone});
+    arr.push({field: "Applied for Grade", value: application.appliedForGrade});
+    arr.push({field: "Status", value: application.status});
+
+    return arr;
+  }
+
+  const tableFormattedData = convertObject(application)
+
+  return (
+    <Layout>
+      <div >
+        <Head>
+          <title>Student Applications Details</title>
+          <link rel="icon" href="../../../public/favicon.ico"/>
+        </Head>
+
+        <main >
+          <Toast  id="decision_toast"
+                  ref={decisionToast}
+                  position="top-right"
+                  onHide={() => router.reload()}
+          />
+          <h1 className={styles.title}>
+            Student Applications Details
+          </h1>
+
+          <Divider />
+
+          <div className="card">
+            <p><Button className="p-button-link">
+              <Link href ="/">
+                <a>Home Page</a>
+              </Link>
+            </Button></p>
+
+            <p><Button className="p-button-link">
+              <Link href ="/registration">
+                <a>Registration Home Page</a>
+              </Link>
+            </Button></p>
+          </div>
+
+          <Divider />
+
+          <div className="card">
+            <DataTable
+              value={tableFormattedData}>
+              <Column field="field" header="Registration ID"/>
+              <Column field="value" header="Name"/>
+            </DataTable>
+          </div>
+
+          <Divider />
+
+          <div style={{ margin: "auto", marginBottom: "1em" }}>
```

Comment:
styles should be written in css files?

Reply:
@ishmum123 bhaia, kindly let us know if these should be moved to separate css files

Reply:
Let's leave it here for the time being... we should get back to this

Reply:
Okay bhai. In that case, would you merge the PR if agreed by lubna apu

Reply:
okay vaiya